docs/developer/services/explorer_service.rst

```diff
@@ -93,18 +90,53 @@ The explorer API provides the following endpoints.
 
   Retrieve the metadata of a node with ``id``. Valid for all nodes.
 
+**GET /graph/node/:id/edges**                    
```

Comment:
I'm not sure this conforms to the API spec -- shouldn't the nouns be plural? I'm guessing this might affect other endpoints so leave until API refactor?